.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,8 +10,7 @@ See also #23
 
 ## Checklist
 
-👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈
-
+- [ ] [Sign off your commits](https://loopback.io/doc/en/contrib/code-contrib.html) with DCO (Developer Certificate of Origin)
```

Comment:
@dhmlau I'm sorry that I misread the checklist initially when I proposed to use a verb style. Maybe the following is better:

```
- [ ] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
```